data/json/construction/walls.json

```
@@ -1408,4 +1408,4 @@
     "pre_terrain": "t_wall_wood",
     "post_terrain": "t_wood_wall_y"
   }
-]
+]
```

```suggestion
]

```

I know it is the least of your problems. Sorry if I got your hopes up.